spec/spec-helper.coffee

```diff
@@ -172,41 +172,47 @@ jasmine.useRealClock = ->
 addCustomMatchers = (spec) ->
   spec.addMatchers
     toBeInstanceOf: (expected) ->
-      notText = if @isNot then " not" else ""
-      this.message = => "Expected #{jasmine.pp(@actual)} to#{notText} be instance of #{expected.name} class"
+      beOrNotBe = if @isNot then "not be" else "be"
+      this.message = => "Expected #{jasmine.pp(@actual)} to #{beOrNotBe} instance of #{expected.name} class"
       @actual instanceof expected
 
     toHaveLength: (expected) ->
       if not @actual?
         this.message = => "Expected object #{@actual} has no length method"
         false
       else
-        notText = if @isNot then " not" else ""
-        this.message = => "Expected object with length #{@actual.length} to#{notText} have length #{expected}"
+        haveOrNotHave = if @isNot then "not have" else "have"
+        this.message = => "Expected object with length #{@actual.length} to #{haveOrNotHave} length #{expected}"
         @actual.length is expected
 
     toExistOnDisk: (expected) ->
-      notText = this.isNot and " not" or ""
-      @message = -> return "Expected path '" + @actual + "'" + notText + " to exist."
+      toOrNotTo = this.isNot and "not to" or "to"
+      @message = -> return "Expected path '#{@actual}' #{toOrNotTo} exist."
       fs.existsSync(@actual)
 
     toHaveFocus: ->
-      notText = this.isNot and " not" or ""
+      toOrNotTo = this.isNot and "not to" or "to"
       if not document.hasFocus()
         console.error "Specs will fail because the Dev Tools have focus. To fix this close the Dev Tools or click the spec runner."
 
-      @message = -> return "Expected element '" + @actual + "' or its descendants" + notText + " to have focus."
+      @message = -> return "Expected element '#{@actual}' or its descendants #{toOrNotTo} have focus."
       element = @actual
       element = element.get(0) if element.jquery
       element is document.activeElement or element.contains(document.activeElement)
 
     toShow: ->
-      notText = if @isNot then " not" else ""
+      toOrNotTo = this.isNot and "not to" or "to"
       element = @actual
       element = element.get(0) if element.jquery
-      @message = -> return "Expected element '#{element}' or its descendants#{notText} to show."
+      @message = -> return "Expected element '#{element}' or its descendants #{toOrNotTo} show."
       element.style.display in ['block', 'inline-block', 'static', 'fixed']
 
+    toBePath: (expected) ->
```

Comment:
> toSamePath perhaps to indicate there is some level of fuzzyness given the normalization on both sides.

`toSamePath` sounds somewhat weird to me, as `expect(...).toSamePath(...)` doesn't read like a valid sentence. It seems like `toEqualPath` might be a better name:

1. It indicates _value equality_.
2. We use the `toEqual` variant elsewhere in the codebase. 
3. It reads nicely when writing assertions.

Ideally I think another solution would be to not use strings to represent paths, and just have a different object (e.g. `Path`) that encapsulates this behavior, so that we can just use `toEqual`. That seems like a high-impact breaking change, though, and therefore it seems like introducing a new matcher for this use-case could be the most pragmatic path.